logistic_consignee: split the module

gurneyalex opened the pull request:
bits are now split in sale_addresses, purchase_addresses,
purchase_requisition_addresses and stock_addresses.
- [x] https://github.com/OCA/purchase-workflow/pull/110
- [x] https://github.com/OCA/stock-logistics-transport/pull/6
- [x] https://github.com/OCA/stock-logistics-transport/pull/4

yvaucher committed:
[FIX] propagation from LR to CE of consignee_shipping_id

gurneyalex committed:
fix shipment from/dest locations for donations

gurneyalex committed:
logistic_consignee: split the module

bits are now split in sale_addresses, purchase_addresses,
purchase_requisition_addresses and stock_addresses.

gurneyalex commented:
superseeds #140

gurneyalex commented:
superseeds #139

gurneyalex committed:
fix view reference

gurneyalex committed:
always call super() in _origin_address

gurneyalex committed:
follow module renaming in stock-logistics-transport

jgrandguillaume commented:
I've tested that and it works fine, So :+1: when build are green

lepistone commented:
@gurneyalex now that dependencies are merged, could you delete the commit changint `.travis.yml` ?

gurneyalex commented:
rebased

yvaucher commented on logistic_order_donation_shipment/model/stock_move.py:
```diff
@@ -0,0 +1,38 @@
+                 'picking_id.group_id.procurement_ids.purchase_id.partner_id')
+    @api.one
+    def _get_ship_addresses(self):
+        super(StockMove, self)._get_ship_addresses()
```
I couldn't find where was this super method defined. I searched for it in stock-logistics-transport but it's not there.

gurneyalex replied:
It comes from https://github.com/OCA/stock-logistics-workflow/blob/8.0/stock_shipment_management/model/stock_move.py#L88 

(and without this call the tests are red).

yvaucher replied:
Ok then we will probably need another PR as this method will disappear in https://github.com/OCA/stock-logistics-transport/pull/4 (move from `stock-logistics-workflow` to `stock-logistics-transport`)

gurneyalex replied:
added the missing dependency. I'll update the PR to work with that changed merged too.

gurneyalex committed:
logistic_order_donation_shipment: no longer required

with the new implementation of stock_shipment_management this module is no
longer required. I renamed it with the _tests suffix and updated the
description to make it clear that it contains only tests

yvaucher commented:
Failing on test:

```
2015-04-09 13:35:44,356 23762 ERROR openerp_test openerp.addons.logistic_order_donation_shipment_test.tests.test_ship_addresses: FAIL
```

gurneyalex commented:
yvaucher: relaunched the build after olivier dony merged https://github.com/odoo/odoo/pull/4993 -> green :fireworks:
